test/message/encryptMessage.spec.js

```diff
@@ -102,3 +102,113 @@ test('it can encrypt a message and decrypt it unarmored using session keys along
     t.is(decrypted, 'Hello world!');
     t.is(verified, VERIFICATION_STATUS.SIGNED_AND_VALID);
 });
+
+test('it can encrypt and decrypt a message with session key without setting returnSessionKey', async (t) => {
+    const decryptedPrivateKey = await decryptPrivateKey(testPrivateKeyLegacy, '123');
+    const sessionKey = {
+        data: new Uint8Array([
+            197,
```

Comment:
I know that the linter wants it like this, but I think allocating 32 lines of code for a session key is pretty ugly :) Perhaps you could convert it to hex (in the code) and then do `data: openpgp.util.hex_to_Uint8Array('...')`